crates/core/src/utils.rs

```diff
@@ -84,23 +79,13 @@ pub fn check_dotfiles(files: &[PathBuf]) -> bool {
 
 /// Get the type of URL from a dependency URL.
 ///
-/// Git URLs are identified by the presence of the `git@github.com` or `git@gitlab.com` prefix, as
-/// well as HTTPS URLs which have the `github.com` or `gitlab.com` domain and a trailing `.git` in
-/// their path.
+/// Git URLs are identified by the presence of the `git@` prefix, as
+/// well as HTTPS URLs which have the trailing `.git` in their path.
 pub fn get_url_type(dependency_url: &str) -> Result<UrlType, DownloadError> {
-    if GIT_SSH_REGEX.is_match(dependency_url) {
+    if dependency_url.starts_with("git@") {
```

Comment:
I'm wondering if we should add a requirement that this ends with `.git`. Also, how likely is it that other servers use a different username than `git`?

Reply:
we can just introduce new flag for git deps to the cli crate